integrations/tensorflow/e2e/BUILD

```diff
@@ -212,3 +213,38 @@ iree_e2e_test_suite(
         "//integrations/tensorflow/bindings/python/pyiree/tf/support",
     ],
 )
+
+iree_e2e_test_suite(
+    name = "mobile_bert_squad_tests",
+    backends_to_srcs = {
+        "tf": ["mobile_bert_squad_test.py"],
+        "tflite": ["mobile_bert_squad_test.py"],
+    },
+    reference_backend = "tf",
+    tags = [
+        "manual",
+        "nokokoro",
+        "notap",
```

Comment:
So the tests don't pass?

Reply:
The tests are slow and flaky (because they involve downloading weights). Same as `:vision_external_tests`.